src/light.scss

```diff
@@ -0,0 +1,8 @@
+@import '~bootstrap/scss/bootstrap';
+@import "~bootstrap-icons/font/bootstrap-icons.css";
+
+body, * {
+  margin: 0;
+  border: 0;
+  box-sizing: border-box;
+}
```

Comment:
I feel like we don't need a "light.css" file at all.
Couldn't this be put inside the regular CSS file?

Reply:
For sure, in the last change I did that.